jekyll/_cci2/config-intro.md

```diff
@@ -111,11 +111,14 @@ Every code base and project is different.  That’s okay.  We like diversity.  T
 
 {% highlight yaml %}
 version: 2.1
+
+orbs:
+  browser-tools: circleci/browser-tools@1.1.0
 jobs:
   build:
     # pre-built images: https://circleci.com/docs/2.0/circleci-images/
     docker:
-      - image: circleci/node:14-browsers
+      - image: cimg/node:17.2-browsers
     steps:
       - checkout
       - run:
```

Comment:
If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`